mm2src/mm2_main/src/lp_swap/swap_watcher.rs

```diff
@@ -377,7 +418,7 @@ impl State for RefundTakerPayment {
             &transaction.tx_hex(),
             1,
             false,
-            watcher_ctx.data.taker_payment_lock + WAIT_FOR_TAKER_REFUND + 3600,
+            watcher_ctx.data.swap_started_at + watcher_ctx.data.lock_duration + WAIT_FOR_TAKER_REFUND + 3600,
```

Comment:
`wait_for_confirmations` doesn't seem required here. Watcher responsibility is to broadcast the tx. Little it can do if tx inclusion in block is delayed.